dev/index.js

```diff
@@ -67,12 +65,7 @@ $(function () {
     .on({
       "/": function () {
         console.log("Home");
-        $("#main")
-          .empty()
-          .append(htmlFrags.index);
-        i18next.changeLanguage(window.lang);
-        jqueryI18next.init(i18next, $);
-        $("body").localize();
+        router.navigate('/en/home');
```

Comment:
is this always going to english ? should it not be `router.navigate(window.lang + '/home')` ?